core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -57,6 +59,9 @@
 
   public static final String REQUEST_TAGS_PROP_KEY = "_request_tags";
 
+  private static final int PERCENTILE_NUM_BUCKETS = 100;
+  private static final double PERCENTILE_MAX_LATENCY_IN_MS = 10 * 1000;
```

Comment:
How about:

```
  private static final double PERCENTILE_MAX_LATENCY_IN_MS = TimeUnit.SECONDS(10).toMillis();
```

10 seconds still seems high to me, and I might prefer a max of 5 seconds with a 50ms granularity. After all, anything in the 99p (5 seconds or more) is still bad performance.

Reply:
re: latencies, we should make sure there aren't any cross DC use cases that could have larger latencies than we tend to think of with single DC cases. e.g., it is possible to run schema registry in single global cluster mode cross DC, in which case a write request in one DC might need to be forwarded across DCs, wait for Kafka write, and come back across DCs. 5s still wouldn't be great, but for that setup it might be acceptable for some operations.

Reply:
In that case, would it be acceptable to have a max of 10s with 200 bins then? We would still have a 50 ms granularity. Is that a better compromise?

Reply:
tbh, i don't have strong opinions here. i was just calling out an example where such high latency might not be considered crazy since the thread seemed to presume that less than ideal out-of-DC possible latencies were unreasonable.

it's reasonable to consider defaults/common case as well and default for those. most use cases are in-DC. if we want to default for those and figure out how to customize/extend later if needed, i'm fine with that.